src/main/java/zmaster587/advancedRocketry/dimension/DimensionProperties.java

```diff
@@ -1760,12 +1734,12 @@ public void writeToNBT(NBTTagCompound nbt) {
 		nbt.setBoolean("isNative", isNativeDimension);
 		nbt.setBoolean("isGasGiant", isGasGiant);
 		nbt.setBoolean("hasRings", hasRings);
-		nbt.setInteger("sealevel", sealevel);
+		nbt.setInteger("seaLevel", sealevel);
```

Comment:
This was the only NBT name problem I found, I think I misread something else

Reply:
I'll change it back

Reply:
Already did it in my PR.